Package.swift

```diff
@@ -20,6 +21,10 @@ let package = Package(
       name: "TokamakDOM",
       targets: ["TokamakDOM"]
     ),
+    .library(
+      name: "TokamakShim",
+      targets: ["TokamakShim"]
+    ),
```

Comment:
This is awesome! Why not call it just `Tokamak`?

Reply:
It could be named just `Tokamak`, I'm only worried that it would hide the actual purpose of the module, which does re-exporting based on the platform. This exporting logic may not work for all cases though. As soon as we have multiple renderers, I could see some people using `TokamakDOM` together with `TokamakHTML` for pre-rendering for example. Or an iOS/macOS app that uses `TokamakHTML` for rendering static websites. In the latter cases you'd import both `SwiftUI` and `TokamakHTML`. I wonder if users would expect an HTML renderer if they import just `Tokamak` on iOS/macOS, while `TokamakShim` points to the platform-per-renderer switching more clearly in my opinion.

IDK, this is just my logic, which may not reflect the actual reality. What do you think @j-f1 @carson-katri?

Reply:
I agree with that logic.